quarkus/service/src/test/java/org/apache/polaris/service/quarkus/IcebergExceptionMapperTest.java

```diff
@@ -52,4 +68,17 @@ void fileIOExceptionMapping(RuntimeException ex, int statusCode) {
       assertThat(response.getEntity()).extracting("message").isEqualTo(ex.getMessage());
     }
   }
+
+  /**
+   * Creates a mock of the Azure-specific HttpResponse object, as it's quite difficult to construct
+   * a "real" one.
+   *
+   * @param statusCode
+   * @return
+   */
+  private static HttpResponse mockAzureResponse(int statusCode) {
+    HttpResponse res = mock(HttpResponse.class);
+    when(res.getStatusCode()).thenReturn(statusCode);
+    return res;
+  }
```

Comment:
What makes it specific to Azure?

Reply:
Azure has its own `com.azure.core.http.HttpResponse` class.

Reply:
I see. Sorry, I missed that :sweat_smile: